Resources/public/apps/adminApp/user/admin-user.html

```diff
@@ -57,7 +57,7 @@
     </article>
     <article class="grid--item">
       <box heading="{{ 'role.heading' | translate }}">
-        <div class="button-icon-link has-spacing-after" ng-click="showAddRoleModal()" ng-if="baseCanUpdate(user)">
+        <div class="button-icon-link has-spacing-after" ng-click="showAddRoleModal()" ng-if="baseCanUpdateRoles(user)">
```

Comment:
I think this should be 

ng-if="baseCanUpdateRoles(baseCurrentUser)"

instead, since we are checking if the logged in user has permission to change roles,
not if the user being edited can change roles.

Reply:
Makes sense :D I've updated the check to use baseCurrentUser